### ISSUE TITLE
is there have a pretrain model

### ISSUE THREAD

**Issue by jcjavaismgood** [TianzhongSong/PSMNet-TensorFlow#3]
it is strange that loss no restrain,i will appreciate it if u could give me advice or share your train weights of KITTI datesets ,thanks a lot!
